src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -726,13 +726,19 @@ class Settings extends Component {
     this.setState({
       enterAsSend: isInputChecked,
     });
+    setTimeout(() => {
+      this.handleSubmit();
```

Comment:
Why setTimeout? You can simply call it in callback function. Always avoid using setTimeout